.github/workflows/main.yml

```
@@ -17,20 +17,27 @@ on:
 
 jobs:
   tests:
-    name: ${{ matrix.os }} / ${{ matrix.python-version }}
+    name: ${{ matrix.os }} / ${{ matrix.python-version }} ${{ matrix.suffix }}
     runs-on: ${{ matrix.os }}-latest
     continue-on-error: ${{ matrix.experimental }}
     strategy:
       matrix:
         os: [Ubuntu, MacOS, Windows]
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
         experimental: [false]
         bootstrap-args: [""]
+        suffix: [""]
         include:
           - os: Ubuntu
-            python-version: "3.10.0-alpha - 3.10.0"
+            python-version: "3.10"
             experimental: true
             bootstrap-args: "--git https://github.com/python-poetry/poetry.git"
+            suffix: (HEAD)
+          - os: Ubuntu
+            python-version: "3.11-dev"
+            experimental: true
+            bootstrap-args: "--git https://github.com/python-poetry/poetry.git"
+            suffix: (Python Dev)
```

```suggestion
            suffix: (HEAD)
```

`HEAD` should be sufficient here as the version is already `3.11-dev`